helper.pl

```diff
@@ -244,10 +257,12 @@ sub process_makefiles {
   }
 
   # update OBJECTS + HEADERS in makefile*
-  for my $m (qw/ makefile makefile.shared makefile_include.mk makefile.msvc makefile.unix makefile.mingw /) {
+  for my $m (qw/ makefile makefile.shared makefile_include.mk makefile.msvc makefile.unix makefile.mingw sources.cmake /) {
     my $old = read_file($m);
     my $new = $m eq 'makefile.msvc' ? patch_file($old, $var_obj)
-                                    : patch_file($old, $var_o);
+            : $m eq 'sources.cmake' ? make_sources_cmake(bsd_glob("*.c"))
```

Comment:
I guess we should also include the header files?

Reply:
ping @mrpelotazo

Reply:
Yes.
I  would use two separate variables for pub/priv headers so only the pub ones are passed to install.